src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1015,8 +1015,14 @@ def vmss_set(**kwargs):
 
 def update_acs(resource_group_name, container_service_name, new_agent_count):
     client = _compute_client_factory()
-    instance = client.container_service.get(resource_group_name, container_service_name)
+    instance = client.container_services.get(resource_group_name, container_service_name)
     instance.agent_pool_profiles[0].count = new_agent_count
-    return client.container_service.create_or_update(resource_group_name,
-                                                     container_service_name, instance)
-
+    return client.container_services.create_or_update(resource_group_name,
+                                                      container_service_name, instance)
+
+def list_container_services(client, resource_group_name=None):
+    ''' List Container Services. '''
+    svc_list = client.container_services.list_by_resource_group(
```

Comment:
@brendandburns, add a test step [here](https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py#L1270). I recently added [similar testing](https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/tests/test_webapp_commands.py#L23) for webapp.
Since people are waiting for this change, I went ahead and merged it. You can catch up the test coverage in the next PR